src/utils/AccessibleButton.ts

```diff
@@ -13,7 +13,7 @@ export enum ArrowDirection {
 
 export class AccessibleButton {
   private element: Dom;
-  private hasLabel = true;
+  private hasLabelOrTitle = true;
```

Comment:
despite the fact that I added the `withoutLabel` function originally, I also had a moment of confusion when reading the name of this variable.

Perhaps it could be renamed `labelOrTitleRequired`, `mustBeDescribed`, or something else of the sort?

Reply:
Will change for labelOrTitleRequired